spec/spec_helper.rb

```diff
@@ -0,0 +1,42 @@
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'log_decorator'
+require "log4r"
```

Comment:
True. I only used it because it's what I've used in the past and I have boilerplate initialization code. I'll try to replace it.